### PR TITLE
[MFTF] Improve AdminLocalized test to be stable

### LINKED ISSUE #1346: [MFTF] Fix AdminAdobeStockLocalizationTest
### Steps to reproduce (*)
Run AdminAdobeStockLocalizationTest

### Expected result (*)
The test should pass

### Actual result (*)
The test is not able to find the keyword "Индустрия" on the page
image

### PR DESCRIPTION
**Pull request by Nazar65**
### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#1346: [MFTF] Fix AdminAdobeStockLocalizationTest
2. ...

### Manual testing scenarios (*)
N/A Green build :heavy_check_mark:

### CODE REVIEW

File: AdobeStockImageAdminUi/Test/Mftf/Test/AdminAdobeStockLocalizationTest.xml [hunk 1]
```diff
@@ -43,12 +43,11 @@
...
-        <!-- The following assertion/requirement on the specific keyword we expect may not be stable as we have no guarantee on the exact keyword to use as the image could be random -->
-        <actionGroup ref="AssertAdminAdobeStockImageKeywordActionGroup" stepKey="assertKeywordLocalized">
-            <argument name="keyword" value="машина"/>
+        <actionGroup ref="AssertAdminAdobeStockKeywordLocalizedActionGroup" stepKey="assertKeywordLocalized">
+             <argument name="stringToBeVerified" value="{{AdminAdobeStockImagePreviewSection.firstKeyword}}"/>
         </actionGroup>
-        <actionGroup ref="AssertAdminAdobeStockImagePreviewAttributeVisibleActionGroup" stepKey="assertCategoryLocalized">
-            <argument name="attributeName" value="Индустрия"/>
+        <actionGroup ref="AssertAdminAdobeStockKeywordLocalizedActionGroup" stepKey="assertCategoryLocalized">
+             <argument name="stringToBeVerified" value="{{imagePreviewCategoryText}}"/>
```
Comment (gabrieldagama) [inline AdobeStockImageAdminUi/Test/Mftf/Test/AdminAdobeStockLocalizationTest.xml]: Wouldn't that be {{AdminAdobeStockImagePreviewSection. imagePreviewCategoryText}} here instead?

File: AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AssertAdminAdobeStockKeywordLocalizedActionGroup.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertAdminAdobeStockKeywordLocalizedActionGroup">
+        <arguments>
+            <argument name="stringToBeVerified" type="string"/>
+        </arguments>
+        <annotations>
+            <description>Verify that string is localized by regexp pattern with RU-ru locale</description>
+        </annotations>
+        <assertRegExp stepKey="checkThatKeywordLocalized">
+            <expectedResult type="string">/^[\p{L}]+$/u</expectedResult>
```
Comment (sivaschenko) [inline AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AssertAdminAdobeStockKeywordLocalizedActionGroup.xml]: This pattern looks to be matching any language. A quick research pointed me to this regex pattern for Cyrillic letters: `[\x{0400}-\x{04FF}]+`

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Hi @Nazar65 thanks for the pull request! I love the solution, however, please see my comment regarding the pattern

**Review by gabrieldagama — CHANGES_REQUESTED** [review]
Hi @Nazar65, thanks for your pull request, please follow below my comments.

**Comment by sivaschenko** [comment]
@Nazar65 I've updated the branch, please unskip the test

File: AdobeStockImageAdminUi/Test/Mftf/Test/AdminAdobeStockLocalizationTest.xml (hunk 1 above)
Comment (sivaschenko) [reply AdobeStockImageAdminUi/Test/Mftf/Test/AdminAdobeStockLocalizationTest.xml]: yep, and we need to grab the value first

Comment (Nazar65) [reply AdobeStockImageAdminUi/Test/Mftf/Test/AdminAdobeStockLocalizationTest.xml]: thanks :+1:

**Comment by Nazar65** [comment]
@sivaschenko  @gabrieldagama  thank for suggestion, all done :heavy_check_mark:

**Comment by Nazar65** [comment]
@magento run  Functional Tests CE Functional Tests B2B

**Comment by engcom-Golf** [comment]
@magento run Functional Tests CE Functional Tests B2B

**Comment by engcom-Golf** [comment]
@magento run Functional Tests CE

**Comment by sivaschenko** [comment]
StoreFrontRecentlyComparedAtWebsiteLevelTest failure is not related to the changes

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**